### PR TITLE
feat: Add ability to override region via ENV var

### PR DESCRIPTION
**Pull request by vdmgolub**

### CODE REVIEW

File: internal/providers/terraform/parser.go [hunk 1]
```diff
@@ -327,6 +332,28 @@ func parseTags(resourceType string, v gjson.Result) map[string]string {
...
+
+	switch providerPrefix {
+	case "aws":
+		region = config.AWSOverrideRegion
+	case "azurerm":
+		region = config.AzureOverrideRegion
+	case "google":
+		region = config.GoogleOverrideRegion
+	default:
+		log.Debugf("Unsupported provider %s", providerPrefix)
```
Comment (hugorut) [inline internal/providers/terraform/parser.go:347]:
```suggestion
		logging.Logger.Debugf("Unsupported provider %s", providerPrefix)
```

File: internal/providers/terraform/parser.go [hunk 2]
```diff
@@ -327,6 +332,28 @@ func parseTags(resourceType string, v gjson.Result) map[string]string {
...
+	}
+
+	if region != "" {
+		log.Debugf("Overriding region (%s) for %s", region, addr)
```
Comment (hugorut) [inline internal/providers/terraform/parser.go:351]:
```suggestion
		logging.Logger.Debugf("Overriding region (%s) for %s", region, addr)
```

File: internal/providers/terraform/parser.go [hunk 3]
```diff
@@ -327,6 +332,28 @@ func parseTags(resourceType string, v gjson.Result) map[string]string {
 	}
 }
 
+func overrideRegion(addr string, resourceType string, config *config.Config) string {
+	region := ""
+	providerPrefix := strings.Split(resourceType, "_")[0]
```
Comment (hugorut) [inline internal/providers/terraform/parser.go:337]:
```suggestion
	providerPrefix := strings.Split(resourceType, "_")
	if len(providerPrefix) == 0 {
	    return ""
	}
```

so we don't panic with invalid provider strings (vscode)

Comment (vdmgolub) [reply internal/providers/terraform/parser.go:337]: Going to do the same in other methods 👍

**Review by hugorut — APPROVED** [review]

**Pull request merged**